EIPS/eip-1690.md

```diff
@@ -0,0 +1,174 @@
+---
+eip: 1690
+title: Contract Mortability Standard
+author: Felix Franz (@Felix-Franz)
+discussions-to: https://github.com/ethereum/EIPs/issues/1690
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-01-11
+requires: 173
+---
+
+## Simple Summary
+
+A standard that allows the owner of a contract to completely remove it.
+
+## Abstract
+
+For removing a smart contract the solidity function `selfdestruct(address);` needs to be called from inside the Contract.
+This function allowes to provide a address where the ether currently owned by the smart contract will be transferred before deleting the contract.
+It should be called from outside the contract by the owner only using a public function.
```

Comment:
```suggestion
Defines two methods (`kill()` and `kill(address)`) whose purpose is to trigger a deletion of the contract and liquidation of any owned assets when called.
```
This all sounds like motivation rather than abstract.  The abstract should be a terse human readable form of the specification.